flink-libraries/flink-table/src/test/java/org/apache/flink/table/api/java/batch/TableEnvironmentTest.java

```diff
@@ -209,28 +206,20 @@ public void testRegisterTableFromOtherEnv() throws Exception {
 	@Test
 	public void testAsFromTuple() throws Exception {
```

Comment:
I think we still need the from tuple, from pojo, and toTuple, toPojo etc. tests. There are no other tests that test if the input and output conversion works correctly.  Can you create `DataSetScanITCase` in `org.apache.flink.table.runtime.dataset`? The overall goal is to reduce ITCases such that we only test the "runtime" classes `DataSetXXX` and `DataStreamXXX`.